beacon-chain/core/helpers/committee.go

```diff
@@ -348,6 +349,12 @@ func UpdateCommitteeCache(state *stateTrie.BeaconState, epoch uint64) error {
 
 // UpdateProposerIndicesInCache updates proposer indices entry of the committee cache.
 func UpdateProposerIndicesInCache(state *stateTrie.BeaconState, epoch uint64) error {
+	// The cache uses the block root at the last epoch slot as key. (e.g. for epoch 1, the key is root at slot 31)
+	// Which is the reason why we skip genesis epoch.
+	if epoch <= params.BeaconConfig().GenesisEpoch {
```

Comment:
just make it
```
if epoch == params.BeaconConfig().GenesisEpoch 
```
unsigned integers cant be lower than zero.

Reply:
True, but is GenesisEpoch always 0?